baseplate/context/memcache/lib.py

```diff
@@ -152,16 +159,17 @@ class PickleFlags(object):
 def decompress_and_unpickle(key, serialized, flags):
     """Deserialization method compatible with pickle_and_compress().
 
+    .. warning:: Please don't use this! Unless you really need backwards
+        compatibility with already stored picked values you should use the JSON
```

Comment:
This might take some wrangling to read more clearly, unless you had another way to phrase this.

> already-stored, picked values

I may have mis-read the intent with the above suggestion. Big take-away is that this warning is ambiguous to someone who isn't familiar with the plumbing!